shared/xccdf/services/base.xml

```diff
@@ -117,6 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
+<platform idref="cpe:/o:virtualization:baremetal"/>
```

Comment:
@radzy Watson can correct me if I am wrong but the root Benchmark is already marked for RHEL7, marking groups or rules by baremetal effectively makes them applicable to RHEL7 AND baremetal only. We can do the same thing with other products.